tests/openbb_terminal/stocks/technical_analysis/test_volatility_model.py

```diff
@@ -0,0 +1,41 @@
+# IMPORTATION STANDARD
+
+# IMPORTATION THIRDPARTY
+import pandas as pd
+import pytest
+
+# IMPORTATION INTERNAL
+from openbb_terminal.common.technical_analysis import volatility_model
+
+# pylint: disable=W0621
+# pylint: disable=W0613
+
+MODELS = volatility_model.VOLATILITY_MODELS
+MOCK_DATA = pd.read_csv(
+    "tests/openbb_terminal/stocks/technical_analysis/csv/test_volatility_model/test_cones_df.csv",
```

Comment:
This will break on windows

Reply:
Which part breaks on Windows? Does it need to be called, `MOCK_STOCK_DF`?

from `test_ta_controller.py`:
```
MOCK_STOCK_DF = pd.read_csv(
    "tests/openbb_terminal/stocks/technical_analysis/csv/test_ta_controller/stock_df.csv",
    index_col=0,
)
```

Reply:
windows doesnt read `/`.  Im fixing